docs: add verawood rbac docs (#1497)
openedx/docs.openedx.org: source/community/release_notes/verawood/verawood_console.rst

.. _Verawood Roles and Permissions Console:

> Roles and Permissions Console

Reviewer: ```suggestion
Introducing More Granular Team Management
```

I'm going to be making a lot of suggestions to make this more aligned with the language we use around release notes (promotion, marketing). Honestly this is why we prefer to write these initially in a wiki.

Reviewer: Ack. I'm realizing now you have two release notes. I don't think that's correct. We only need one for this feature, it's confusing to have this split across two release notes. I'm going to pull the text into a wiki page and we can remove the release notes from this PR and handle them separately.

Reviewer: I will make a PR for this.

Author: I'll remove the release notes from this PR.

############################################

   .. image:: /_images/educator_how_tos/console_course_team_members.png
      :alt: The Team Members tab of the Roles and Permissions console, listing role assignments across libraries and courses
      :width: 800

> The Verawood release upgrades the Roles and Permissions console,
> letting you manage user access across multiple parts of the platform
> from a single view.

Reviewer: ```suggestion
Easily manage user access across multiple parts of the platform
from a single view. The Verawood release extends the :ref:`administrative console <Ulmo console>`,
introduced in the previous release, with additional new roles that can be applied
across Studio.
```

> You can now assign a role to one or multiple users, and apply it to
> one or multiple spaces, all in a single action. Verawood also adds
> organization-level assignment. Granting a role to a user at the
> organization level covers not only what exists today, but everything
> created in that organization afterward.

Reviewer: ```suggestion
You can now assign two new roles, :ref:`Course Admin or Course Staff <Course Authoring Roles>` to one or multiple users, and apply it
at multiple levels (full instance, a specific organization, or a specific course), all in a single action.
Granting a role to a user at the organization level covers both currently-created courses
as well as any courses created in that organization afterward.
```

> This is the same console already used for libraries. With Verawood,

Reviewer: ```suggestion
This is the same console already used for managing permissions over :ref:`Content Libraries <Add users to Libraries>`. With Verawood,
```

course authoring also lives here too, if enabled — see

> :ref:`Verawood Course Authoring Roles`.

Reviewer: ```suggestion
:ref:`Course Authoring Roles`.
```

.. _Verawood Console Available:

What's Available in Verawood
*****************************

* **Unified interface**: shows all role assignments in one place,
  filterable by role, organization, and scope.

* **Assign Role wizard**: assign a role to multiple users and multiple
  scopes in a single action.

* **Organization-level assignment**: grant a role to a user across an
  entire organization, including scopes created later.

* **User audit view**: shows a user's roles across every course and
  library you have permission to manage.

* **Filtered entry points**: the console opens filtered to the context
  you came from, unfiltered from the Studio home page, or filtered to
  a single course or library when opened from there.

Not Affected by This Release
*****************************

* Course content and how it is authored. The console changes how course
  teams and library teams are managed, not what authors create.

* :ref:`Legacy Libraries <Legacy Content Libraries Overview>` that have

>   not yet been migrated to Content Libraries.

Reviewer: Note for myself: Investigate what our support level is now for Verawood

* LMS features outside of role assignment, such as grading, cohorts,

>   groups, and discussion forums.

Reviewer: I don't understand what this means

* Whether course authoring roles are available for a given course,
  which depends on the Course Authoring waffle flag; see
  :ref:`Verawood Course Authoring Roles`.

.. seealso::

   :ref:`Verawood Course Authoring Roles` (release notes)

   :ref:`Manage Course Authoring Roles` (how-to)

   :ref:`Add users to Libraries` (how-to)

   :ref:`Guide to Course Team Roles` (reference)

**Maintenance chart**

+--------------+-------------------------------+----------------+--------------------------------+
| Review Date  | Working Group Reviewer        |   Release      |Test situation                  |
+--------------+-------------------------------+----------------+--------------------------------+

> | [DATE]       | [REVIEWER]                    |   Verawood     | [PASS/FAIL]                    |

Reviewer: Please fill out before merge.
